http/src/main/resources/init-resources/init-actions.sh

```diff
@@ -135,6 +136,30 @@ function apply_start_user_script() {
 # .. after jupyter notebook start
 # END
 STEP_TIMINGS=($(date +%s))
+
+
+## Installs Google Cloud Ops Agent that is now required for Datapoc 2.2.X ###
```

Comment:
This is the main change in addition to updating docker compose, and that sneaky change in external IP assignment behavior.

It is annoying that the new log agent does not come pre-built into the dataproc image itself, but the install and setup was not too too bad in the end